_includes/footer.html

```diff
@@ -8,6 +8,16 @@
 <div class="d-flex flex-wrap justify-content-center footer-head">
   {{ site.data.snippets.footer.license[page.lang] | markdownify }}{{ site.data.snippets.footer.financial[page.lang] | markdownify }}
 </div>
+ 
+<div class="d-flex flex-wrap justify-content-center footer-head">
+ <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+  <input type="hidden" name="cmd" value="_s-xclick" />
+  <input type="hidden" name="hosted_button_id" value="7BGHUZRVS4LYL" />
+  <input type="image" src="https://www.paypalobjects.com/en_GB/i/btn/btn_donate_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
+  <img alt="" border="0" src="https://www.paypal.com/en_GB/i/scr/pixel.gif" width="1" height="1" />
+</form>
+<a href="https://www.patreon.com/bePatron?u=11816614" data-patreon-widget-type="become-patron-button">Become a Patron!</a><script async src="https://c6.patreon.com/becomePatronButton.bundle.js"></script>
```

Comment:
confirm that this is all that's needed to display Patron button

Reply:
apologies. the footer buttons got left in after my initial playing. If we are going to keep them in (and no need not to) by saying "confirm that this is all that's needed to display Patron button" do you mean the changes you've made work?

Reply:
I have made no changes...?

I do not like the footer buttons - I think the lesson buttons are fine. If you intend to remove the footer buttons, then do so.

Reply:
Sorry. I didn't get your first message. I'll remove the footers. Aren't my favourite either.